### PR TITLE
Fix UI data pull

### PR DESCRIPTION
**Pull request by leogoldberg**
add qr code printing
add edit/delete for parking lots
clean up ui a bit

### CODE REVIEW

File: src/components/App.jsx [hunk 1]
```diff
@@ -22,16 +22,66 @@ import {
 class App extends Component {
     constructor(props) {
         super(props)
-        this.state = { name: '' }
+        this.state = { name: '', allParkingMap: {} }
         this.onNameChange = this.onNameChange.bind(this);
     }
 
+    deleteLot = (id) => {
```
Comment (JasonXD-CS) [inline src/components/App.jsx:30]: function binding in constructor same as `onNameChange`?

File: src/components/App.jsx [hunk 2]
```diff
@@ -22,16 +22,66 @@ import {
...
+        const allParkingMap = Object.assign({}, this.state.allParkingMap)
+        delete allParkingMap[id]
+        this.setState(
+            { allParkingMap })
+    }
+
+    updateLot = (id, name, number, rate, address) => {
```
Comment (JasonXD-CS) [inline src/components/App.jsx:37]: binding

File: src/components/App.jsx [hunk 3]
```diff
@@ -22,16 +22,66 @@ import {
...
+                    number,
+                    rate,
+                    address
+                }
+            }
+
+        }))
+    }
+
+    addLot = (id, parkingLot) => {
```
Comment (JasonXD-CS) [inline src/components/App.jsx:54]: binding

**Review by JasonXD-CS — APPROVED** [review]
Otherwise looks good

**Comment by jorawarc** [comment]
Mind changing the `EventService` URL to point back to the production server in this PR too

**Pull request merged**